README.md

```diff
@@ -58,14 +58,21 @@ $ export HUB_NAME=<hub name>
 $ make deploy-with-external-etcd
 ```
 
-### Option 3: Start multicluster-controlplane in local
+### Option 3: Start multicluster-controlplane as a local binary
 
 ```bash
 $ make all
 ```
-<b>Access the controlplane</b>
 
-The kubeconfig file of the controlplane is in the dir `hack/deploy/certs/kubeconfig`.
+### Option 4: Start multicluster-controlplane with embedded etcd on KinD cluster
+
+```bash
+$ make setup-e2e
```

Comment:
the target name `setup-e2e` is confusing here.

Reply:
Okay, I  have added another `make deploy-on-kind` target to prevent confusion.